pages/EntryListPage/index.js

```diff
@@ -0,0 +1,14 @@
+import JournalEntryList from "@/components/JournalEntryList";
+import Link from "next/link";
+
+export default function EntryListPage() {
+  return (
+    <>
+      {" "}
```

Comment:
Is the {" "} there on purpose? If yes, ignore my comment :😄

Reply:
they come automatically without function, i remove them, thx